views/networking/load_balancer/show.erb

```diff
@@ -0,0 +1,136 @@
+<% @page_title = @lb[:name] %>
+
+<div class="space-y-1">
+  <%== render(
+    "components/breadcrumb",
+    locals: {
+      back: "#{@project_data[:path]}/load-balancer",
+      parts: [
+        %w[Projects /project],
+        [@project_data[:name], @project_data[:path]],
+        ["Load Balancer", "#{@project_data[:path]}/load-balancer"],
+        [@lb[:name], "#"]
+      ]
+    }
+  ) %>
+  <%== render("components/page_header", locals: { title: @lb[:name] }) %>
+</div>
+<div class="grid gap-6">
+  <!-- Detail Card -->
+  <%== render(
+    "components/kv_data_card",
+    locals: {
+      data: [
+        ["ID", @lb[:id]],
+        ["Name", @lb[:name]],
+        ["Connection String", @lb[:hostname], { copieble: true }],
+        [
+          "Private Subnet",
+          "<a href='#{@project_data[:path]}/location/#{@lb[:subnet][:location]}/private-subnet/#{@lb[:subnet][:name]}' class='text-rose-500 hover:underline'>#{@lb[:subnet][:name]}</a>",
```

Comment:
`Serializers::PrivateSubnet` has `include_path` path option available. You can use the path as `@lb[:subnet][:path]`